app/styles/components/tf-toggle-button.css

```diff
@@ -0,0 +1,96 @@
+/* 
+  Variables 
+*/
+
+.c-tf-toggle-button {
+  /* toggle button */
+  --tf-tb-min-width-primary: 54px; 
+  --tf-tb-min-height-primary: 32px; 
+  --tf-tb-min-width-secondary: 40px; 
+  --tf-tb-min-height-secondary: 24px; 
+  --tf-tb-border: none;
+  --tf-tb-box-shadow: inset 0 0 2px 0 rgba(0,0,0,0.12);
+  --tf-tb-border-radius: 20px;
+
+  /* inner circle */
+  --tf-tb-ic-size-primary: 26px;
+  --tf-tb-ic-size-secondary: 19px;
+  --tf-tb-ic-border-radius: 20px;
+  --tf-tb-ic-box-shadow: 0 1px 2px 0 rgba(0,0,0,.25);
+}
+
+/*
+  Base button
+*/
+
+.c-tf-toggle-button {
+  border: var(--tf-tb-border);
+  border-radius: var(--tf-tb-border-radius);
+  -moz-box-shadow:    var(--tf-tb-box-shadow);
+  -webkit-box-shadow: var(--tf-tb-box-shadow);
+  box-shadow:         var(--tf-tb-box-shadow);
+  background-color: var(--x-color-gray-1);
+  text-align: left;
```

Comment:
Agreed and aware of this. Just wanted to get something out easiest as possible until I get some input from you / designers.